.github/workflows/publish.yml

```
@@ -20,3 +20,13 @@ jobs:
       with:
         command: publish
         args: --all-features
+    - name: slack notification (the job has failed)
+      if: failure()
+      run: |
+        curl -s -X POST -H "Content-Type: application/json" -d '${{ env.SLACK_TEXT }}' ${{ secrets.SLACK_CORE_TOOLS_TEAM_URL }}
+      env:
+        SLACK_TEXT: '{"attachments":[{
+        "pretext": "`transit_model Publish` requires your attention!",
+        "text":" :warning: Publish failed!","color":"#D00000",
+        "fields":[{"title":"Action URL","value": "https://github.com:/${{ github.action_path }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"}]}
```

(If you want to test it, maybe just use `echo` instead of `curl` to avoid spamming slack :canned_food:)

It does work, but it doesn't look like a legit url :thinking: 
Maybe more:
```suggestion
        "fields":[{"title":"Action URL","value": "https://github.com${{ github.action_path }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"}]}
```

Also, I don't know if what's suggested [here](https://docs.github.com/en/actions/reference/environment-variables) would work (probably better if possible, as it's straight from official doc):  `$GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID`

---

i change url and test it. Done